internal/commands/open_test.go

```diff
@@ -0,0 +1,144 @@
+package commands_test
+
+import (
+	"context"
+	"fmt"
+	"hellper/internal/bot"
+	"hellper/internal/commands"
+	filestorage "hellper/internal/file_storage"
+	"hellper/internal/log"
+	"hellper/internal/model"
+	"testing"
+
+	"github.com/slack-go/slack"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+type openCommandFixture struct {
+	testName             string
+	expectError          bool
+	errorMessage         string
+	ctx                  context.Context
+	mockLogger           log.Logger
+	mockClient           bot.Client
+	mockRepository       model.Repository
+	mockFilestorage      filestorage.Driver
+	triggerID            string
+	mockDialogSubmission bot.DialogSubmission
```

Comment:
Not a request.
I like to break these fields in fields related to the test, fields that are parameters of the function, mocks. Just an idea.

Reply:
Nice! I can improve it in v2